install/gcp/installer/util/gcloud.py

```diff
@@ -803,13 +805,20 @@ def check_vm_init_status(vm_name, zone):
 
     check_script_executed = 'tail -n1 /tmp/deployment.log'
 
-    _, out, _ = utils.run_command(
+    _, out, err = utils.run_command(
         ['gcloud', 'compute', 'ssh', vm_name,
          '--zone', zone, '--command', check_script_executed, '--quiet'])
     # --quiet flag is needed to eliminate the prompting for user input
     # which will hang the run_command function
     # i.e. It will create a folder at ~/.ssh and generate a new ssh key
 
+    if err:
+        print('Error occurred when sshing to the VM, will skip waiting for '
+              'the instance to be initialized and proceed with the rest of '
+              'the installation.')
```

Comment:
I would add a message to tell the user that they might need to verify the VM is actually initialized themselves.

```
'Error occurred when sshing to the VM.  Unable to verify the VM is fully initialized.'
'This is a non-fatal error, but you should verify the VM is fully initialized before use (add how).'
'Will skip waiting for the instance to be initialized and proceed with the rest of '
'the installation.'
```

Reply:
Updated.